cpp/example_code/elasticfilesystem/create_file_system_with_modeled_exceptions.cpp

```diff
@@ -0,0 +1,120 @@
+
+//snippet-sourcedescription:[Upgrade AWS SDK for C++ to version 1.8 to build create_file_system_with_modeled_exception.cpp. This example demonstrates how to get modeled exception from CreateFileSystem operation outcome.]
+//snippet-keyword:[C++]
+//snippet-sourcesyntax:[cpp]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon Elastic File System]
+//snippet-service:[elasticfilesystem]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[]
+//snippet-sourceauthor:[AWS]
+
+/*
+   Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This file is licensed under the Apache License, Version 2.0 (the "License").
+   You may not use this file except in compliance with the License. A copy of
+   the License is located at
+
+    http://aws.amazon.com/apache2.0/
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations under the License.
+*/
+#include <iostream>
+#include <aws/core/Aws.h>
+#include <aws/core/utils/Outcome.h>
+#include <aws/core/utils/logging/LogLevel.h>
+#include <aws/elasticfilesystem/EFSClient.h>
+#include <aws/elasticfilesystem/model/CreateFileSystemRequest.h>
+#include <aws/elasticfilesystem/model/DescribeFileSystemsRequest.h>
+#include <aws/elasticfilesystem/model/DeleteFileSystemRequest.h>
+#include <aws/elasticfilesystem/model/FileSystemAlreadyExists.h>
+
+using namespace Aws;
+using namespace Aws::Http;
+using namespace Aws::Client;
+using namespace Aws::EFS;
+using namespace Aws::EFS::Model;
+
+static const char FILE_SYSTEM_CREATION_TOKEN[] = "file-system-creation-token-test";
+
+Aws::String CreateFileSystem(const Aws::EFS::EFSClient efsClient)
```

Comment:
You might want to consider adding standardized usage documentation just before each function declaration, describing the function's purpose, any prerequisites, inputs, and any return values.